care/utils/evaluators/interpretation_evaluator.py

```
@@ -0,0 +1,204 @@
+        if "age" in required_keys:
+            date_delta = (
+                datetime.fromisoformat(effective_datetime).date()
+                - patient.date_of_birth
+            )
+            context["age"] = date_delta.days / 365.25
+
+        if "applies_to" in required_keys:
+            from care.emr.tagging.base import (
+                PatientFacilityTagManager,
+                PatientInstanceTagManager,
+            )
```

Ruff (0.12.2)

`import` should be at the top-level of a file

(PLC0415)